crates/biome_service/src/file_handlers/astro.rs

```diff
@@ -69,98 +67,87 @@ impl ExtensionHandler for AstroFileHandler {
     fn capabilities(&self) -> Capabilities {
         Capabilities {
             enabled_for_path: EnabledForPath {
-                formatter: Some(javascript::formatter_enabled),
-                search: Some(javascript::search_enabled),
-                assist: Some(javascript::assist_enabled),
-                linter: Some(javascript::linter_enabled),
+                formatter: Some(html::formatter_enabled),
+                search: Some(html::search_enabled),
+                assist: Some(html::assist_enabled),
+                linter: Some(html::linter_enabled),
```

Comment:
Let's refrain from making public facing changes for now. We haven't made enough progress to do that yet -- if we were to merge this we would regress in functionality.